repositories.yaml

```diff
@@ -845,12 +845,15 @@ applications:
         app_channel: release
         additional_dependencies:
           - glean-js
+          # It also uses the Rust SDK.
+          - glean-core
       - v1_name: mozilla-vpn-android
         app_id: org.mozilla.firefox.vpn
         app_channel: release
         description: Mozilla VPN (Android)
         additional_dependencies:
           - glean-core
+          # Extra dependency to have `android_sdk_version` recorded.
```

Comment:
This isn't directly related, but I figured since I was adding a comment for the other dependency I might as well add a clarifying comment here after the [discussion in probe-scraper#552](https://github.com/mozilla/probe-scraper/pull/552/files#r1099860766).